include/RED4ext/Scripting/Natives/gameIGameSystem-inl.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/Scripting/Natives/gameIGameSystem.hpp>
+#endif
+
+#include <RED4ext/Scripting/Natives/Generated/game/SaveLock.hpp>
+#include <RED4ext/Scripting/Natives/Generated/world/RuntimeScene.hpp>
+
+RED4EXT_INLINE void RED4ext::game::IGameSystem::OnWorldAttached(RED4ext::world::RuntimeScene* aScene)
```

Comment:
Same comment as in [IUpdatableSystem-inl.hpp](https://github.com/WopsS/RED4ext.SDK/pull/75/files#diff-5720cff5fa06b528ec192fe670fc5be5bb1267072d16b6bf8537021dcfc2fe95).

Reply:
It also has an empty default implementation. It makes sense, not all game systems need full lifecycle.

Reply:
I would have expected that the interface is pure and then the child will declare these empty.